src/wccp.cc

```diff
@@ -357,6 +364,9 @@ wccpAssignBuckets(void)
                   buf,
                   wab_len + WCCP_BUCKETS + cache_len,
                   0);
+    // XXX: setup fd_table[] properly so we can use FD_WRITE_METHOD()
+    fd_table[sock].bytesWritten(wab_len + WCCP_BUCKETS + cache_len);
```

Comment:
Same here: Missing protection from comm_udp_send() failures. GitHub will not let me post a specific correction for this one.